examples/flexvolume/README.md

```diff
@@ -0,0 +1,58 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
You should add content from your https://github.com/kubernetes/kubernetes/pull/13840#issuecomment-142411692 to the doc as well.